Marlin/src/lcd/menu/menu_ubl.cpp

```diff
@@ -73,21 +73,14 @@ static void _lcd_mesh_fine_tune(PGM_P const msg) {
   }
 }
 
-void lcd_limbo() {
-  ui.currentScreen = []{};
-  ui.defer_status_screen();
-}
-
-float lcd_mesh_edit() {
-  lcd_limbo();
-  ui.refresh(LCDVIEW_CALL_REDRAW_NEXT);
-  _lcd_mesh_fine_tune(GET_TEXT(MSG_MESH_EDITOR));
-  return mesh_edit_value;
-}
+//
+// Called external to the menu system to acquire the result of an edit.
+//
+float lcd_mesh_edit() { return mesh_edit_value; }
 
 void lcd_mesh_edit_setup(const float &initial) {
   mesh_edit_value = mesh_edit_accumulator = initial;
-  lcd_limbo();
```

Comment:
Doesn't this go below  the next line?

Reply:
It goes away with this patch. Why do you ask?